### DISCUSSION TITLE
fi.set_model_parameters() not working for FLORIS 3.1.1

### DISCUSSION THREAD

**Discussion by Diederikvb** [NatLabRockies/floris#458] [Q&A] [answered]
How do I change wake parameters in FLORIS 3.1.1? I've been changing the source code, but there must be an easier way to change the wake parameters (like in 2.4) right? For my purpose I'm looking to try a list of different parameter combinations, instead of changing the .YAML  file.  

Edit: Applied #399, worked so far.

**Comment by rafmudaf** [comment, marked as answer]
@Diederikvb I see you found your answer in #399. If that worked, please mark this one as the answer for the benefit of other people who come here.
